samples/src/Microsoft.Azure.IIoT.App/src/Program.cs

```diff
@@ -31,7 +51,8 @@ public class Program {
                     .UseContentRoot(Directory.GetCurrentDirectory())
                     .UseStartup<Startup>()
                     .UseKestrel(o => o.AddServerHeader = false)
-                    .UseIISIntegration());
+                    .UseIISIntegration())
+                .UseSerilog();
```

Comment:
Indentation.

Reply:
This is according to `Serilog.AspNetCore` [instructions](https://github.com/serilog/serilog-aspnetcore#instructions).